roles/mariadb/tasks/main.yml

```diff
@@ -1,29 +1,26 @@
-# Install a docker based MariaDB.
+# Install  MariaDB.
 ---
 - name: Make MariaDB/MySQL settings volume.
   file:
     path: "{{ item }}"
     state: directory
-    mode: '0777' # This is broken, insecure, oops. 
+    mode: '0750'
     owner: root
     group: root
   with_items:
-      - '/srv/mariadb/lib/mysql'
-      - '/srv/mariadb/etc/mysql'
-      - '/srv/mariadb/etc/mysql/conf.d'
-  notify:
-    - restart_mysql
+      - '/lib/mysql'
+      - '/etc/mysql'
+      - '/etc/mysql/conf.d'
   become: true
 
-- name: Deploy MariaDB/MySQL config file.
-  copy:
-    src: 'files/my.cnf'
-    dest: '/srv/mariadb/etc/mysql/conf.d/my.cnf'
-    mode: '0660'
-    owner: root
-    group: root
+- name: Yum install mariadb-server.
+  yum:
+    name:
+      - mariadb-server
+    state: latest
+    update_cache: yes
   notify:
-    - restart_mysql
+    - start_mysql
```

Comment:
You will need ```restart``` as opposed to simply ```started``` to ensure an updated version will be active after an ```mariadb-server``` update by ```yum```. Without the restart the old version will remain active in memory.